src/App.tsx

```diff
@@ -1,203 +1,92 @@
-/* eslint-disable jsx-a11y/label-has-associated-control */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { USER_ID, getTodos } from './api/todos';
+import { TodoList } from './components/TodoList';
+import { HeaderInput } from './components/HeaderInput';
+import { Footer } from './components/Footer';
+import classNames from 'classnames';
+import { Errors, SelectedTasks, Todo } from './types/Types';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState<Errors | null>(null);
+  const [selectedTasks, setSelectedTasks] = useState<SelectedTasks>(
+    SelectedTasks.All,
+  );
+
+  useEffect(() => {
+    getTodos()
+      .then(setTodos)
+      .catch(error => {
+        setErrorMessage(Errors.Load);
+        throw error;
```

Comment:
That's a bit rude :)) 

What exactly should happen here? 
Setting the error would do just fine.